packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6895,26 +6903,65 @@ export default class Meeting extends StatelessWebexPlugin {
   /**
    * Publishes a stream.
    *
-   * @param {LocalStream} stream to publish
    * @param {MediaType} mediaType of the stream
+   * @param {LocalStream} stream to publish
    * @returns {Promise}
    */
-  private async publishStream(stream: LocalStream, mediaType: MediaType) {
-    if (this.isMultistream && this.mediaProperties.webrtcMediaConnection) {
-      await this.sendSlotManager.publishStream(mediaType, stream);
+  private async publishStream(mediaType: MediaType, stream?: LocalStream) {
+    if (!stream) {
+      return;
```

Comment:
Should we also throw an error stating - `stream not found` or something similar?

Reply:
no, because publishStream() is sometimes called with undefined stream in normal use cases - see for example setLocalAudioStream()